### PR TITLE
Move content from ./src/popup/scripts/ to ./src/popup/

### PR DESCRIPTION
**Pull request by smoralesd**
#### Description of changes

Our `popup` folder is the only one where we have a `scripts` folder inside. This is usual at the root level (similar to our `src` folder) but not necessary on the lower level folder. 
Moving the content inside `scripts` folder one level up with this PR.

#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000 **N/A**
- [ ] Added relevant unit test for your changes. (`npm run test`) **N/A**
- [ ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage` **N/A**
- [x] Ran precheckin (`npm run precheckin`)
- [ ] Added screenshots/GIFs for UI changes.

### CODE REVIEW

File: src/DetailsView/components/left-nav/nav-link-handler.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import { autobind } from '@uifabric/utilities';
-
+import * as React from 'react';
```
Comment (Shobhit1) [inline src/DetailsView/components/left-nav/nav-link-handler.ts]: Do we need `React` for a `ts` file?

File: src/tests/unit/tests/popup/scripts/components/header.test.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
```
Comment (peterdur) [inline src/tests/unit/tests/popup/scripts/components/header.test.tsx]: Should we also remove the scripts folder from the test tree?
src/tests/unit/tests/popup/**scripts**/components/header.test.tsx

Comment (smoralesd) [reply src/tests/unit/tests/popup/scripts/components/header.test.tsx]: Good catch... totally forgot about the tests :(

File: src/DetailsView/components/left-nav/nav-link-handler.ts (hunk 1 above)
Comment (smoralesd) [reply src/DetailsView/components/left-nav/nav-link-handler.ts]: done... we should have a closer look at this though

File: src/tests/unit/tests/popup/scripts/components/header.test.tsx (hunk 2 above)
Comment (smoralesd) [reply src/tests/unit/tests/popup/scripts/components/header.test.tsx]: @peterdur Updated

**Pull request merged**